daemons/based/based_callbacks.c

```diff
@@ -727,69 +727,22 @@ forward_request(xmlNode *request)
     pcmk__xe_remove_attr(request, PCMK__XA_CIB_DELEGATED_FROM);
 }
 
-static gboolean
-send_peer_reply(xmlNode * msg, xmlNode * result_diff, const char *originator, gboolean broadcast)
+static void
```

Comment:
bool

Reply:
Nothing checks it.

I botched a rebase last night where I changed the `return` lines to avoid the compile error.  I thought I went back and fixed it, but different branch.